apps/cli/src/service-container/service-container.ts

```
@@ -254,13 +257,13 @@ export class ServiceContainer {
     } else if (process.env.BITWARDENCLI_APPDATA_DIR) {
       p = path.resolve(process.env.BITWARDENCLI_APPDATA_DIR);
     } else if (process.platform === "darwin") {
-      p = path.join(process.env.HOME, "Library/Application Support/Bitwarden CLI");
+      p = path.join(process.env.HOME ?? "", "Library/Application Support/Bitwarden CLI");
     } else if (process.platform === "win32") {
-      p = path.join(process.env.APPDATA, "Bitwarden CLI");
+      p = path.join(process.env.APPDATA ?? "", "Bitwarden CLI");
     } else if (process.env.XDG_CONFIG_HOME) {
       p = path.join(process.env.XDG_CONFIG_HOME, "Bitwarden CLI");
     } else {
-      p = path.join(process.env.HOME, ".config/Bitwarden CLI");
+      p = path.join(process.env.HOME ?? "", ".config/Bitwarden CLI");
```

I thought about continuing to let it crash here instead of hiding the error if these are `undefined`, but wasn't sure. Please let me know if anyone thinks thats a better way to handle these values.

---

For the non-windows cases we could also use `os.homedir()`, which has a fallback when the HOME env variable is not defined: https://nodejs.org/api/os.html#oshomedir.

---

Cool! That would be more involved testing than I'd like for this change though, so going to hold off on that for now.